packages/shared/src/components/modals/ReputationPrivilegesModalTrigger.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement, useEffect } from 'react';
+import { useActions } from '../../hooks';
+import { ActionType } from '../../graphql/actions';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { LazyModal } from './common/types';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+const threshold = 10;
```

Comment:
Shouldn't this be 250?
(I thought I commented this, but can't see)

Reply:
yes, just seen it locally while testing 😆